compass/landice/tests/humboldt/run_model.py

```diff
@@ -0,0 +1,191 @@
+from compass.model import run_model, make_graph_file
+from compass.step import Step
+
+
+class RunModel(Step):
+    """
+    A step for performing forward MALI runs as part of Humboldt test cases.
+
+    Attributes
+    ----------
+    mesh_type : str
+        The resolution or mesh type of the test case
+
+    velo_solver : str
+        The velocity solver used for the test case
+
+    calving_law : str
+        The calving law used for the test case
+
+    suffixes : list of str, optional
+        a list of suffixes for namelist and streams files produced
+        for this step.  Most steps most runs will just have a
+        ``namelist.landice`` and a ``streams.landice`` (the default) but
+        the ``restart_run`` step of the ``restart_test`` runs the model
+        twice, the second time with ``namelist.landice.rst`` and
+        ``streams.landice.rst``
+
+    mesh_file : str
+        The name of the mesh file being used
+
+    forcing_file : str
+        The name of the forcing file being used
+
+    """
+    def __init__(self, test_case, velo_solver, mesh_type,
+                 name='run_model',
+                 calving_law=None,
+                 damage=None,
+                 face_melt=False,
+                 subdir=None, cores=1,
+                 min_cores=None, threads=1, suffixes=None):
+        """
+        Create a new test case
+
+        Parameters
+        ----------
+        test_case : compass.TestCase
+            The test case this step belongs to
+
+        velo_solver : {'sia', 'FO', 'none'}
+            The velocity solver setting to use for this test case
+
+        calving_law: {'none', 'floating', 'eigencalving',
+                      'specified_calving_velocity', 'von_Mises_stress',
+                      'damagecalving', 'ismip6_retreat'}, optional
+            The calving law setting to use for this test case. If not
+            specified, set to 'none'.
```

Comment:
I wish this worked.  The docstring doesn't parse correctly:
![image](https://user-images.githubusercontent.com/4179064/164713559-00e3bd65-84dc-4df0-856f-493084bbef18.png)
Instead, I'd suggest:
```suggestion
        calving_law: str, optional
            The calving law setting to use for this test case. If not
            specified, set to 'none'. Valid values are: 'none', 
            'floating', 'eigencalving', 'specified_calving_velocity', 
            'von_Mises_stress', 'damagecalving', 'ismip6_retreat'}
```

Reply:
Fixed in ecfafe4cdb998425c88b87d97bb4ec84b8a53208